package.json

```diff
@@ -211,6 +207,7 @@
     "sass-lint": "^1.12.1",
     "sass-loader": "^7.1.0",
     "scroll-into-view-if-needed": "^1.5.0",
+    "semantic-release": "^15.10.7",
```

Comment:
I think you also want to change package.version to something like `0.0.0-semantically-released`

Reply:
right, done.